airbyte-integrations/connectors/source-mysql/build.gradle

```diff
@@ -8,7 +8,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.35.11'
     features = ['db-sources']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
not sure if we need to merge this change (and the one below) ?

Reply:
Will revert this when cdk is published and is toggled